src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -416,12 +435,12 @@ def CheckInterference(self, sas_uut_active_grants, margin_db,
         neighbor_distances=self.neighbor_distances,
         threshold=hard_threshold
     )
-    # TODO(sbdt): could do early stop as soon as one fails, although I would expect
-    # the criteria to be changed into checking 99.x% of success instead of 100%.
     pool = mpool.Pool()
     result = pool.map(checkPointInterf, self.protected_points)
-    max_diff_interf = max(result)
 
+    self.__PrintStatistics(result, margin_db, self.name, channel)
+
+    max_diff_interf = max([r.max_difference for r in result])
```

Comment:
use : max_diff_interf = max(r.max_difference for r in result)

This will avoid creation of the list, as it will use iterator.